rest/config.go

```diff
@@ -68,6 +68,7 @@ const (
 
 // JSON object that defines the server configuration.
 type ServerConfig struct {
+	SecureCookieOverride       *bool                    `json:"secure_session_cookies,omitempty"` // Override cookie secure flag
```

Comment:
Does it make sense to move this config option to be scoped at the db-level, like the `SessionCookieHttpOnly` and `SessionCookieName` options?